fish/PKGBUILD

```diff
@@ -11,8 +11,8 @@ msys2_references=(
 )
 license=('GPL2')
 groups=()
-depends=('bc' 'gcc-libs' 'gettext' 'libiconv' 'libpcre2_16' 'man-db' 'ncurses')
-makedepends=('doxygen' 'gcc' 'gettext-devel' 'intltool' 'libiconv-devel' 'ncurses-devel' 'pcre2-devel' 'cmake')
```

Comment:
Why was libiconv-devel removed?

Reply:
fish used to depend on iconv for Unicode support, but replaced it with a modified version of https://www.bsdua.org/libbsdua.html which is vendored internally in 2014. I'll drop it from depends as well, thanks for the pickup!

Reply:
It still links against libiconv: https://github.com/msys2/MSYS2-packages/pull/4812/checks#step:10:950

Reply:
hm, it seems ntldd and Dependencies seem to now show libiconv being linked, unlike ldd. So maybe ldd is buggy somehow there.

Reply:
I got confused by ntldd only showing direct deps that I forgot about ldd showing all of them, sorry for the confusion.